src/filemanager.cpp

```diff
@@ -1182,8 +1183,16 @@ static void CreateDirectoryRecursively(PJ_CONTEXT *ctx,
 
 // ---------------------------------------------------------------------------
 
-std::string pj_context_get_user_writable_directory(PJ_CONTEXT *ctx,
-                                                   bool create) {
+/** Get the PROJ user writable directory for datumgrid files.
+ *
+ * @param ctx PROJ context, or NULL
+ * @param create Create the directory if it does not exist already.
+ * @return The path to the PROJ user writable directory.
+ * @since 7.1
+*/
+
+const char *proj_context_get_user_writable_directory(PJ_CONTEXT *ctx,
```

Comment:
Doxygen complains about this function and the other not being documented. This is because currently they must be within a //! @cond Doxygen_Suppress block. So either move them in a place in the file close to already exported method, or surround them with "//! @endcond" and "//! @cond Doxygen_Suppress"

Reply:
That is good to know - I made another attempt at what I think will fix it. :crossed_fingers: